File: test/fixtures/node-axios/post_with_data_raw.js
import axios from 'axios';

const response = await axios.post(
  'http://localhost:28139/post',
  'msg1=wow&msg2=such&msg3=@rawmsg',
  {
    headers: {
      'Content-Type': 'application/x-www-form-urlencoded'
    }
  }
);

File: test/fixtures/r-httr2/post_with_data_raw.r
library(httr2)

request("http://localhost:28139/post") |>
  req_method("POST") |>
  req_body_raw(
    "msg1=wow&msg2=such&msg3=@rawmsg",
    type = "application/x-www-form-urlencoded"
  ) |>
  req_perform()
